packages/mui-lab/src/internal/pickers/PickersPopper.tsx

```diff
@@ -196,6 +223,8 @@ const PickersPopper = (props: PickerPopperProps) => {
     role,
     TransitionComponent = Grow,
     TrapFocusProps,
+    showTodayButton = false,
```

Comment:
The popper shouldn't have anything to do with it's content. It's just for displaying its content in a popper. The content should be fully controlled by its callsite. So the logic for the today button should be owned by whomever uses `PickersPopper`

Reply:
Okay will look into that and let you know